### PR TITLE
Projects section popup window

### PR DESCRIPTION
**Pull request by Koshao-sys**
- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] You need to use a JavaScript array to store all of the information for all projects.
- [ ] For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- [ ] You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- [ ] After that, you could implement the popup window.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -25,4 +25,216 @@ closeLinks.forEach((closeLink) => {
...
+        linkingList.appendChild(linkList);
+      }
+
+      mainDiv.appendChild(closebtnDiv);
+      mainDiv.appendChild(imgContainer);
+      mainDiv.appendChild(projHeaderdiv);
+      mainDiv.appendChild(listElement);
+      mainDiv.appendChild(paragraph);
+      mainDiv.appendChild(linkingList);
+      popupWindow.appendChild(mainDiv);
```
Comment (AnselemOdims) [inline script.js]: - [ ] At the moment, when you open a project modal for the second time onwards it adds new modal to the already opened modal. Which means the second is two modals in one, third is three and so on. Please kindly fix this.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Koshao-sys , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Koshao-sys** [comment]
@AnselemOdims That bit has really confused me, I can't keep up with my code anymore but let me try to rectify it.

**Comment by Koshao-sys** [comment]
Fixed issue; When you open a project modal for the second time, it adds new modal to the already opened modal. Which means the second is two modals in one, third is three and so on.

**Review by OyePriscilla — APPROVED** [review]
Hi @Koshao-sys 😄, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✔️GitHub flow observed 👍
✔️App worked properly 👍
✔️Made required changes 👍

## Optional suggestions

- [optional] Kindly remove the placeholder from the future features section of the README file, this will make it cleaner. Thanks. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**